example/requirements.txt

```
@@ -0,0 +1 @@
+modelz
```

This is not the TensorChord one
```suggestion
modelz-py
```

---

Fixed.